src/python/artm/_js/ldavis.js

```diff
@@ -0,0 +1,1361 @@
+LDAvis = function(to_select, data) {
```

Comment:
The license is MIT
Noted [here](https://github.com/cpsievert/LDAvis/blob/master/DESCRIPTION)